src/style.css

```
@@ -0,0 +1,29 @@
+html body {
```

It is enough to reset body margin and padding
```suggestion
body {
```